dbus.yaml

```diff
@@ -0,0 +1,82 @@
+package:
+  name: dbus
+  version: 1.14.4
+  epoch: 0
+  description: Freedesktop.org message bus system
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: AFL-2.1 OR GPL-2.0-or-later
+
+secfixes:
+  1.14.4-r0:
+    - CVE-2022-42010
+    - CVE-2022-42011
+    - CVE-2022-42012
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - expat-dev
+      - libsm-dev
+      - libx11-dev
+      - meson
+      - xmlto
+      - bash
+      # - glib-dev # TODO do we need this?  This and downstream packages are compiling fine
```

Comment:
Also not sure about `glib-dev`?

Reply:
This is needed for the libdbus-glib dependency, which is obsolete but still used by some software.

Reply:
ok sounds good, will package glib-dev too

Reply:
for now we could unblock JDK and address TODOs in a followup https://github.com/wolfi-dev/os/issues/49